### PR TITLE
i18n(zh-cn): Update project-structure.mdx

### PR DESCRIPTION
**Pull request by liruifengv**
#### What kind of changes does this PR include?

- Update project-structure.mdx

#### Description

- Add `tsconfig.json`
- Add FileTree

### CODE REVIEW

File: src/pages/zh-cn/core-concepts/project-structure.mdx [hunk 1]
```diff
@@ -3,6 +3,7 @@ layout: ~/layouts/MainLayout.astro
 title: 项目结构
 description: 学习如何用 Astro 构建一个项目。
 ---
+import FileTree from '~/components/FileTree.astro'
 
 你用 `create-astro` CLI 向导所生成的新 Astro 项目中已经包括一些文件和文件夹。其他的则你将自己创建并添加到 Astro 的现有文件结构中。
```
Comment (delucis) [inline src/pages/zh-cn/core-concepts/project-structure.mdx:8]: We recently updated this to remove the hyphen in `create astro`:

```suggestion
你用 `create astro` CLI 向导所生成的新 Astro 项目中已经包括一些文件和文件夹。其他的则你将自己创建并添加到 Astro 的现有文件结构中。
```

See: https://github.com/withastro/docs/commit/7abc2abe98077d7596ebbd38b979b62c5d20ba4b#diff-2f438cd874e276d48f0738cb2eae905c3407ec81207c7293a1bbaa13991bcea3R9

**Review by delucis — COMMENTED** [review]
Thanks @liruifengv! One small additional change needed but this looks great 🙌

**Comment by liruifengv** [comment]
> 

Thanks. There are a few more `create-astro` in the Chinese documentation, I will create a new pr to fix them.

**Review by delucis — APPROVED** [review]
Thanks! If you update `create-astro` elsewhere, it would be good to double check we also have the same usage updated in the `/en/` versions too.

**Pull request merged**